boards/BMS/Src/App/App_Accumulator.c

```diff
@@ -20,6 +20,20 @@ enum AccumulatorMonitorState
     GET_CELL_TEMP_STATE,
 };
 
+struct VoltageStats
+{
+    struct
+    {
+        uint8_t  segment;
+        uint8_t  series_elem;
+        uint16_t voltage;
+    } min, max;
+    uint32_t segment_avg[ACCUMULATOR_NUM_OF_SEGMENTS];
+    uint32_t segment[ACCUMULATOR_NUM_OF_SEGMENTS];
+    uint32_t pack_avg;
+    uint32_t pack;
```

Comment:
Nit: Include voltage in these variable names so it's clear these are all voltages